lib/src/parse/stylesheet.dart

```diff
@@ -3283,16 +3340,19 @@ abstract class StylesheetParser extends Parser {
   // ## Supports Conditions
 
   /// Consumes a `@supports` condition.
-  SupportsCondition _supportsCondition() {
+  ///
+  /// If [inParentheses] is true, the indented syntax will consume newlines
+  /// where a statement otherwise would end.
+  SupportsCondition _supportsCondition({bool inParentheses = false}) {
     var start = scanner.state;
     if (scanIdentifier("not")) {
-      whitespace();
+      whitespace(allowNewlines: true);
```

Comment:
I think we should _only_ allow newlines here when `inParentheses` is true, since we want to eventually treat `@supports` like any other at-rule. Same below.